curriculumBuilder/tests/test_tips.py

```diff
@@ -17,12 +17,12 @@ def test_rendering_basic_tip(self):
         )
         expected = (
             '<div class="admonition content">\n'
-            '<p class="admonition-title" id="content_content-0"><i class="fa fa-mortar-board"></i>Content Corner</p>\n'
-            '<div></div>\n'
-            '<div>\n'
-            '<h3 id="teaching-this-course-as-a-class">Teaching this course as a class?</h3>\n'
-            '<p>Our grade-aligned CS Fundamentals courses use unplugged lessons to build community and introduce tricky computer science concepts, including <strong>events</strong>. Check out the lesson <a href="https://curriculum.code.org/csf-19/coursea/11/" target="_blank">The Big Event Jr.</a> from <a href="https://curriculum.code.org/csf-19/coursea/" target="_blank">Course A</a>!</p>\n'
-            '</div>\n'
+                '<p class="admonition-title" id="content_content-0"><i class="fa fa-mortar-board"></i>Content Corner</p>\n'
```

Comment:
Why did the whitespace change?

Reply:
Oh, the whitespace is just here for readability; it doesn't change the actual content of the variable

Reply:
Figured it was something like this but wanted to double check!